zero_bin/tools/prove_rpc.sh

```diff
@@ -88,6 +91,19 @@ else
     BLOCK_INTERVAL=$START_BLOCK..=$END_BLOCK
 fi
 
+# Print out a warning if the we're using `native` and our file descriptor limit is too low. Don't bother if we can't find `ulimit`.
+if [ $(command -v ulimit) ] && [ $NODE_RPC_TYPE == "native" ]
+then
+    file_desc_limit=$(ulimit -n)
+
+    if [[ $file_desc_limit -lt $RECOMMENDED_FILE_HANDLE_LIMIT ]]
+    then
+        echo "WARNING: Maximum file descriptor limit may be too low to run native mode (current: $file_desc_limit, Recommended: ${RECOMMENDED_FILE_HANDLE_LIMIT}).
+        Consider increasing it with:
+
+        ulimit -s ${RECOMMENDED_FILE_HANDLE_LIMIT}"
```

Comment:
Isn't `-s` only updating the stack size, as opposed to `ulimit -n` which updates the maximum number of open file descriptors?

Reply:
Ahh, good catch! Not sure how that got swapped with `-s`.